app/models/client/efile_security_information.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: client_efile_security_informations
+#
+#  id                 :bigint           not null, primary key
+#  browser_language   :string
+#  client_system_time :string
+#  ip_address         :inet
+#  platform           :string
+#  timezone_offset    :string
+#  user_agent         :string
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  client_id          :bigint
+#  device_id          :string
+#
+# Indexes
+#
+#  index_client_efile_security_informations_on_client_id  (client_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (client_id => clients.id)
+#
+class Client::EfileSecurityInformation < ApplicationRecord
+  belongs_to :client
+
+  def client_system_datetime
```

Comment:
We might want to add a note here as to why we store the time as a string in the db, as this is a bit unexpected (the reason, i _think_, is that our database would attempt to coerce the date object into UTC time and then we'd lose the timezone information that we need on it)